### DISCUSSION TITLE
request features

### DISCUSSION THREAD

**Discussion by zsst10** [aleganza/akuse#109] [General]
can you add a skip oppening feature and the time when the oppening ends like netflix if thats possible and french dub

**Comment by Nansess** [comment]
![Screenshot 2024-07-16 132213](https://github.com/user-attachments/assets/35702502-cbd2-49c7-a4de-7d8592822ca3)
it already has one

**Comment by Y345ez** [comment]
ya you need to use aniskip api to make it like netflix
